src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+.header__container {
+  margin: 0 auto;
+  padding: 10px 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo__container {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  gap: 20px;
+  line-height: 100%;
+  align-items: center;
+  height: 40px;
+}
+
+.nav__item {
+  height: 40px;
+  display: flex;
+  font-family: "Roboto", sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)